<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogNewPart</class>
 <widget class="QDialog" name="dialogNewPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Part</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelNewPartName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartDepartment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartManufacturer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>159</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Manufacturer</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartModel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartCategory">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>109</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartStorage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>92</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartDescription">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>660</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPartStockLimit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>123</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>Stock Limit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPartName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of part</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPartManufacturer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of manufacturer or brand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPartModel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Name of model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNewPartStockLimit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Minimum stock quantity</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineNewPartDescription">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>660</y>
     <width>500</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Information of part</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNewPartCategory">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNewPartStorage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNewPartDepartment">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNewPartAdd">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>840</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNewPartUndo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>840</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
